src/main/default/lwc/gameApp/gameApp.html

```diff
@@ -112,11 +112,11 @@ <h1 class="answer-label">{correctAnswerLabel}</h1>
             </template>
         </div>
     </div>
-    <div class="slds-text-align_center slds-text-color_weak">
+    <!--<div class="slds-text-align_center slds-text-color_weak">
```

Comment:
Please revert that change (we want to keep the footer) and change the URL to `https://github.com/fostive/quiz-host-app`

Reply:
I am sorry, didn't want to push it.